hive/src/test/java/com/netease/arctic/hive/catalog/TestArcticHiveCatalog.java

```diff
@@ -41,4 +56,61 @@ public static Object[] parameters() {
     return new Object[] {new HiveCatalogTestHelper(TableFormat.MIXED_HIVE, TEST_HMS.getHiveConf()),
                          new HiveCatalogTestHelper(TableFormat.ICEBERG, TEST_HMS.getHiveConf())};
   }
+
+  @Test
+  public void testArcticTableRootLocation() throws TException {
+    ArcticCatalog hiveCatalog = getCatalog();
```

Comment:
Add the statement `Assume.assumeTrue(TableFormat.MIXED_HIVE.equals(getTestFormat()))` as the first statement for test method to skip other table format